data/topic-schema/src/main/kotlin/net/corda/schema/Schemas.kt

```diff
@@ -118,13 +118,21 @@ class Schemas {
         companion object {
             const val P2P_OUT_TOPIC = "p2p.out"
             const val P2P_OUT_MARKERS = "p2p.out.markers"
+            const val P2P_OUT_MARKERS_STATE = "p2p.out.markers.state"
```

Comment:
I think we need to add `dlq` as well

(https://github.com/corda/corda-api/pull/408/files)